src/components/AccountDrawer/MiniPortfolio/Activity/parseRemote.tsx

```diff
@@ -78,6 +78,12 @@ const COMMON_CONTRACTS: { [key: string]: Partial<Activity> | undefined } = {
   },
 }
 
+const SPAMMABLE_ACTIVITY_TYPES = [TransactionType.Receive, TransactionType.Mint, TransactionType.Unknown]
+function isSpam({ NftTransfer, TokenTransfer }: TransactionChanges, type: TransactionType): boolean {
+  if (!SPAMMABLE_ACTIVITY_TYPES.includes(type)) return false
```

Comment:
what if a user has willingly swapped a spam token

Reply:
if a user willingly made any action I don't think we should hide that activity, IMO spam is outside action

Reply:
hm but it looks like in this logic here, if a user willingly minted a spam NFT - we still mark it as spam? Just want to keep consistent

Reply:
Good point. I still feel strongly about hiding outside action only, so I think it makes more sense for me to update this to only hide activity where` activity.from !== the user` & the tx contains spam, rather than hide any tx that contains a spam flag